ResearchKit/Common/ORKScaleSliderView.m

```diff
@@ -423,6 +431,14 @@ - (instancetype)initWithFormatProvider:(id<ORKScaleAnswerFormatProvider>)formatP
                                                      options:NSLayoutFormatAlignAllCenterY | NSLayoutFormatDirectionLeftToRight
                                                      metrics:@{@"kMargin": @(kMargin)}
                                                        views:views]];
+            
+            if ([_formatProvider shouldHideValueLabel]) {
+                [self addConstraints:
+                 [NSLayoutConstraint constraintsWithVisualFormat:@"V:[_valueLabel(==0)]"
```

Comment:
I have seen this constraint twice, can you consolidate them? Maybe merge the latest changes from master branch first.